### PR TITLE
feature: AbstractPhpUnitFixer - support attribute detection in docblock insertion

### PR DESCRIPTION
**Pull request by paulbalandan**
Partial fix of #6795

### CODE REVIEW

File: tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php
Comment (Wirone) [inline tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php:1]: Can we add cases where multiple attributes are above the class, please? Here, and other tests related to atteibutes.

**Review by Wirone — CHANGES_REQUESTED** [review]
Nice work! 🍻 But I believe we need more complex cases too.

**Comment by kubawerlos** [comment]
Isn't this a feature, not a bug(fix)?

**Comment by paulbalandan** [comment]
> Isn't this a feature, not a bug(fix)?

Not really sure. Since this was reported as a bug, I noted the PR as a bugfix.

**Comment by kubawerlos** [comment]
> Since this was reported as a bug

What do you mean? Linked issue is a feature request.

**Comment by paulbalandan** [comment]
> > Since this was reported as a bug
> 
> What do you mean? Linked issue is a feature request.

My understanding is it is a bug since the docblock is inserted incorrectly. But you're correct, it is indeed a feature request.

File: tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php
Comment (paulbalandan) [reply tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php:1]: Sure. But for more complex cases I'm not sure what to add. 😂

Comment (Wirone) [reply tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php:1]: Just what I wrote, multiple attributes 🙂. I believe your current implementation don't support cases like this:

```
<?php

#[SimpleTest]
#[AnotherAttribute]
class Test extends TestCase
{
}
```

Comment (paulbalandan) [reply tests/Fixer/PhpUnit/PhpUnitInternalClassFixerTest.php:1]: I believe it is now covered by https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6858/commits/17302b3acbdfce4d7e522f15e2f6fbfe4c6789be

**Review by kubawerlos — APPROVED** [review]

**Review by Wirone — APPROVED** [review]

**Comment by kubawerlos** [comment]
thank you @paulbalandan

**Pull request merged**